ZoKrates/Dockerfile

```diff
@@ -25,21 +24,20 @@ RUN apt-get update && apt-get install -y \
 
 USER zokrates
 
-RUN curl https://sh.rustup.rs -sSf | \
-    sh -s -- --default-toolchain nightly -y
+RUN curl https://sh.rustup.rs -sSf \
+    | sh -s -- --default-toolchain nightly -y
 
 ENV PATH=/home/zokrates/.cargo/bin:$PATH
 
-RUN git clone https://github.com/scipr-lab/libsnark.git $LIBSNARK_SOURCE_PATH
-RUN git clone https://github.com/fleupold/ZoKrates.git $ZOKRATES_SOURCE_PATH
-
-WORKDIR $LIBSNARK_SOURCE_PATH
-RUN git checkout $LIBSNARK_COMMIT
-RUN git submodule update --init --recursive
+RUN git clone https://github.com/scipr-lab/libsnark.git $LIBSNARK_SOURCE_PATH \
+    && cd $LIBSNARK_SOURCE_PATH \
+    && git checkout $LIBSNARK_COMMIT \
+    && git submodule update --init --recursive
 
-WORKDIR $ZOKRATES_SOURCE_PATH
-RUN git checkout $ZOKRATES_COMMIT
-RUN cargo build --release
+RUN git clone https://github.com/fleupold/ZoKrates.git $ZOKRATES_SOURCE_PATH \
+    && cd $ZOKRATES_SOURCE_PATH \
+    && git checkout $ZOKRATES_COMMIT \
+    && cargo build --release
```

Comment:
This step takes a lot of time I believe. We could tag and push a docker image that has performed all steps until here (@gnosis/zokrates-compiled or something like that) and then make this Dockerfile based on that docker image.

```
FROM @gnosis/zokrates-compiled
```